packages/design-system/src/components/N8nSticky/Sticky.vue

```diff
@@ -178,6 +179,11 @@ export default mixins(Locale).extend({
 	},
 	watch: {
 		editMode(newMode, prevMode) {
+			if(newMode === false) {
+				// When exiting the edit mode, attach click listeners to links
+				// in case new links have been added
+				this.$refs.stickyMarkdown.initLinkListeners();
```

Comment:
don't like to couple parent and child components like this by calling child methods directly.. in this case, we should simply listen for all click events on all links

Reply:
Agree.

Reply:
still need to delete it :)